### ISSUE TITLE
Adding/subtracting letters' value at the end of the game (Polish)

### ISSUE THREAD

**Issue by alkamid** [quackle/quackle#5]
In Polish, instead of adding the double of the opponent's rack at the end of the game, we add it to the finishing player's score and subtract it from their opponent's score, e.g.:

> Player1: ZZ -  +0 406
> Player2: S 1L S... +21 404
> Player1: ZZ (ZZ) -2 404
> Player2:  (ZZ) +2 406
> (as described in .gcg format specification, http://www.poslfit.com/scrabble/gcg/)

instead of:

> Player1: ZZ -  +0 406
> Player2: S 1L S... +21 404
> Player2:  (ZZ) +4 408

It doesn't really change the game analysis, but it would be nice if Quackle could handle this, if only for the sake of keeping our records clean.

**Comment by rtolsson** [comment]
+1. We adjust the scores the same way in Swedish Scrabble.

**Comment by jfultz** [comment]
Maybe this should be specced as part of the tile distribution (in the alphabets files) then?

**Comment by alkamid** [comment]
Good idea, as the alphabet files are the only human readable language-specific files.

**Comment by andy-k** [comment]
+1 for this feature but -1 for speccing this as part of the tile distribution. The WESPA Championship 2015 was played to +x/-x instead of +2x.
